scheduler/scheduler.go

```diff
@@ -250,6 +276,7 @@ func BuildDependencyGraph(c client.Interface, sel labels.Selector) (DependencyGr
 			depGraph[child].Requires, depGraph[parent])
 
 		depGraph[child].Meta[parent] = d.Meta
+		depGraph[parent].Meta[parent] = d.Meta
```

Comment:
OK. After discussion I admit, that this is wrong and a result of my misunderstanding. I will correct it.